avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -220,6 +220,8 @@ meshCellSize = 5
 meshCellSizeThreshold = 0.001
 # clean DEMremeshed directory to ensure remeshing if chosen meshCellsize is different from rasters in Inputs/
 cleanRemeshedRasters = True
+# resize files read from Inputs/RASTERS to be resized to extent of DEM as factor x meshCellSize
```

Comment:
maybe write resizeThreshold x meshCellSize to make clear that this is the factor?